studio/components/interfaces/BillingV2/Subscription/Tier/Tier.constants.ts

```diff
@@ -0,0 +1,97 @@
+export interface PricingInformation {
+  id: string
+  new: boolean
+  name: string
+  preface?: string
+  features: string[]
+  footer?: string
+}
+
+// This only contains the meta information of plans like description and features, prices are from the API
+// Also ideally shift this to common folder (e.g packages/data)
+export const SUBSCRIPTION_PLANS: PricingInformation[] = [
+  {
+    id: 'tier_free',
+    new: false,
+    name: 'Free',
+    preface: undefined,
+    features: [
+      'Unlimited API requests',
```

Comment:
These items in the feature array do not match the pricing page (wrong ordering and some items missing) - see `apps/www/pages/pricing/index.tsx`